packages/prover/local_setup.sh

```diff
@@ -6,11 +6,6 @@ mkdir -p build
 npm install -g snarkjs@latest
 pip install -r requirements.txt
 mkdir build && cd build
-gdown "https://drive.google.com/uc?id=1XDPFIL5YK8JzLGoTjmHLXO9zMDjSQcJH"
+gdown "https://drive.google.com/uc?id=1l3mNqFYv-YZc2efFlphFUkoaCnGCxFtE"
```

Comment:
Did you make a new proving key and a verifier contract?
If so, could you update the verifier contract in the contracts package and update the google drive url in the following file as well?
https://github.com/zkemail/ether-email-auth/blob/feat/gpu/packages/contracts/README.md#build-and-test

Reply:
I did not make a new proving key, the zkey is the old one, hence updation of verifier contract is not needed